### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by kickb71**

### CODE REVIEW

**Comment by kickb71** [comment]
The date icon has not be been moved

**Review by kuldeep3 — CHANGES_REQUESTED** [review]
@kickb71 Please describe the PR and add a link to the issue so that reviewer can be aware of what the PR is all about. Right now, it's quite hard to understand what changes are proposed in the PR. Moreover, please resolve the conflicts first and then assign it for review.

**Comment by kickb71** [comment]
I did not know any thing about these conflict and the PR link to this issue is
https://github.com/ColoredCow/portal/issues/811

**Comment by P4NK4J** [comment]
@kickb71 , There are some merge conflicts in this PR, maybe you need to take a pull from the master branch and resolve them. In your previous comment, I think you wanted to share the issue linked with the PR. 

Also, the issue description is for what we want to achieve, and in the PR description, you need to mention what changes you did to achieve the desired functionality (described in the issue).

Let me know if that makes sense, or if you have any questions.

**Comment by kickb71** [comment]
These issues are not from my file which is ready for pull request.

**Comment by P4NK4J** [comment]
Probably you should get an understanding of what merge conflicts are, how they appear and how you can resolve them. Here is a [link](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts) for reference.

**Comment by GauravGusain98** [comment]
@kickb71 
Remember, whenever you make a PR please write a description about the changes you have made and issue you resolved. For reference you can take help from this image 

image

File: app/Sql.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Sql extends Model
+{
+    protected $table = 'hr_applicants';
+    public $timestamps = false;
+    protected $fillable = ["id","name","created_at","updated_at"]; 
+
+}
```
Comment (GauravGusain98) [inline app/Sql.php:14]: This file is not needed as there is already a model for this table. You have to use that model. Here is the path of that model `Modules\HR\Entities\Applicant.php`

File: app/Http/Controllers/DashboardController.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use App\Sql;
+use Carbon\Carbon;
+
+class DashboardController extends Controller
+{
+  function index(){   
+    $today_count = Sql::whereDate('created_at', '=',now())
```
Comment (GauravGusain98) [inline app/Http/Controllers/DashboardController.php:12]: Please use meaningful variable names. Also, we don't use `snake_case` naming convention in Laravel. Please use `camelCase` for naming variables and functions.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/DashboardController.php (hunk 2 above)
Comment (kickb71) [reply app/Http/Controllers/DashboardController.php:12]: Okay Sir i will change

File: app/Sql.php (hunk 1 above)
Comment (kickb71) [reply app/Sql.php:14]: I need this to leave out time stamp of all the data from database

Comment (GauravGusain98) [reply app/Sql.php:14]: I didn't get what you mean by `leave out timestamp of all the data from database`. But whatever the case is, you don't need this file as it is not a good practice to have multiple files/functions for the same purpose. 

If you want to make some changes then you can do that in the `Applicant.php`. Also if you have used this model at other places, then please correct them as well.

Comment (kickb71) [reply app/Sql.php:14]: All the entry in the database is  like this for July 2nd (02-07-2021 10:34:54)  I want to remove last and display as (02-07-2021) in the chart that's why.

**Comment by kickb71** [comment]
@ColoredCow/team-portal-reviewers  This PR is ready for review

File: app/Sql.php (hunk 1 above)
Comment (GauravGusain98) [reply app/Sql.php]: I am not sure what you are trying to do, please describe that thing in the description clearly. 

Also, the approach you are taking is incorrect. Whatever the issue is but a table should not have more than one model.

Comment (kickb71) [reply app/Sql.php]: I have changed it as per you suggested and will update the final out put now!

**Comment by P4NK4J** [comment]
@kickb71 , Not sure why this PR closed after adding the ready-to-review comment. Please explain the purpose of status inconsistencies and closure of this PR.

**Comment by kickb71** [comment]
All my files in tge local is deleted that's why i have created a new one

**Pull request closed without merging**